cmd/ddev/cmd/start_test.go

```diff
@@ -34,7 +34,8 @@ func TestCmdStart(t *testing.T) {
 	// Confirm all sites are running.
 	apps := ddevapp.GetActiveProjects()
 	for _, app := range apps {
-		assert.True(app.SiteStatus() == ddevapp.SiteRunning, "All sites should be running, but %s status: %s", app.GetName(), app.SiteStatus())
+		status, _ := app.SiteStatus()
+		assert.True(status == ddevapp.SiteRunning, "All sites should be running, but %s status: %s", app.GetName(), status)
```

Comment:
This would also be better with showing the statusDesc.

Reply:
I've added statusDesc to the message output but I think it might be a bit confusing, that the value shown in the message differs from the one that is asserted on.
I've changed assert.true to assert.equal as this should provide a better native message on fail as it will show the expected status and the given one instead of just saying "expected true, got false".
I also added an assertion on the statusDesc because... why not?

Reply:
Sorry, I was saying to show BOTH 'status=%v' and statusDesc=%v, not to show statusDesc as status.